src/pages/FlashCards/FlashCards.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import "./FlashCards.css"
+function FlashCards() {
+
+
+    const addFlashCard = () => {
```

Comment:
Since this function will run as the result of an event (an onClick event) it is an event handler. Event handlers have a naming convention - they usually start with the word "handle". So maybe, handleCreateFlashCard or something like that. That being said, I don't think the flashcard creation feature will work this way, so go ahead and remove this function for now. 

I think we can keep this page very simple for now since we are still working out the user flow and basic functionality. 

Here is how I see this page working, feel free to be creative though!
I think the list of available flashcard sets will be on the dashboard. The user will click on a set and be sent to this page, which will hold only the current set of flashcards that the user is studying. There should be a place for the current set of flashcards to be rendered as well as a button to edit the set. Perhaps a button for flipping the card and buttons to move forward and backward through the deck as well. 

No need to write functions for these right now. We will need to wait until we have a set of flashcards to work with, I think. The UI can be put in place though, so go ahead and add that.

I hope that's clear - if not feel free to DM me!

Reply:
Also, don't forget to merge main into this branch as well.